### PR TITLE
feat: add words array to environment

### PR DESCRIPTION
**Pull request by zebolino**
Add the current command to the environment to be able to e.g. display the man page in preview:

```
# Preview man page when completing options
zstyle ':fzf-tab:complete:*:options' fzf-preview 'if [[ -n "${command}" ]]; then man ${command} | bat --color=always --language=man --style=numbers ; fi'
```

### CODE REVIEW

**Comment by zebolino** [comment]
Changed the export to the words array.
As I am new to zsh there might be a better way.
At least my man page preview works with this line:
```
# Preview man page when completing options
zstyle ':fzf-tab:complete:*:options' fzf-preview 'if (( $+commands[${words// */}] )); then man "${words// */}" | bat --color=always --language=man --style=numbers ; fi'
```

**Comment by zebolino** [comment]
Fixed a small typo too.

File: lib/-ftb-fzf [hunk 1]
```diff
@@ -22,6 +22,7 @@ export word=\${(Q)ctxt[word]}
 if (( \$+ctxt[realdir] )); then
   export realpath=\${ctxt[realdir]}\$word
 fi
+export words='${words}'
```
Comment (Aloxaf) [inline lib/-ftb-fzf:25]: You can use `$(typeset -p words)` here

**Comment by zebolino** [comment]
Now words is an array so the man page example can be changed to:
```
# Preview man page when completing options
zstyle ':fzf-tab:complete:*:options' fzf-preview 'if (( $+commands[${words[1]}] )); then man "${words[1]}" | bat --color=always --language=man --style=numbers ; fi'
```

File: lib/-ftb-fzf (hunk 1 above)
Comment (zebolino) [reply lib/-ftb-fzf]: Changed the export to your suggestion.

**Pull request closed without merging**